service/storage_proxy.cc

```diff
@@ -809,8 +809,8 @@ storage_proxy::storage_proxy(distributed<database>& db, storage_proxy::config cf
     , _view_update_handlers_list(std::make_unique<view_update_handlers_list>()) {
     namespace sm = seastar::metrics;
```

Comment:
The distribution above is exponential, so the patch description is incorrect.

Reply:
not entirely, it replaces the exponential distribution we had with something more forgiven near the 1s buckets

Reply:
right, but everything on the way supports it, we can get finer resolution when we need it

Reply:
in that case we can keep the 1.2 distribution that is currently reported, we want to lower the number of buckets from the number that is used to collect the data (90) to a lower number